docs/docs-content/getting-started/aws/setup.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: "Set up Palette"
+title: "Set up Palette with AWS"
+description: "Learn how to set up Palette with AWS."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["getting-started", "aws"]
+---
+
+In this guide, you learn how to set up Palette for use with your AWS cloud account. These steps are required in order to
+authenticate Palette and allow it to deploy host clusters.
+
+## Enablement
+
+The prerequisite steps to getting started with Palette on AWS are as follows.
+
+- Sign up to [Palette](https://www.spectrocloud.com/get-started).
+
+  - Your Palette account role must have the `clusterProfile.create` permission to create a cluster profile. Refer to the
+    [Roles and Permissions](../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile-admin)
+    documentation for more information.
+
+- Sign up to a public cloud account from
+  [AWS](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account).
+
+  - Register the AWS cloud account in Palette. The
```

Comment:
Do we want to send the user away from the tutorial? It might make sense to walk them through a basic static credentials scenario, and if they want to use STS, then link to the how-to. The way I look at it, if they struggle with this part, we might loose the reader for the remainder of the tutorial.

Reply:
Good shout. Used MDX partials to put all instructions in line.